src/include/simeng/arch/aarch64/Instruction.hh

```diff
@@ -283,6 +283,43 @@ enum class InsnType : uint32_t {
   isBranch = 1 << 14
 };
 
+/** Predefined shift values for converting pred-as-counter to pred-as-mask. */
+const uint64_t predCountShiftVals[9] = {0, 1, 2, 0, 3, 0, 0, 0, 4};
```

Comment:
Unless I've missed something, this is used in one location. Why is the data defined as a variable outside of all function scopes?

Reply:
Removed as can calculate it automatically